guides/release-process.md

```diff
@@ -157,7 +153,7 @@ In the following instructions, "X.Y.Z" is used to denote the version of Cypress
 
```

Comment:
under step 4, can we update the example shell snipped to be the full cdn path? 
```
npm publish https://cdn.cypress.io/beta/npm/X.Y.Z/<long sha>/cypress.tgz --tag dev
```

Reply:
65ae48b232